src/SSL/RecommendationClient.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace UKFast\SDK\SSL;
+
+use UKFast\SDK\Client;
+use UKFast\SDK\SSL\Entities\Recommendation;
+
+class RecommendationClient extends Client
+{
+    protected $basePath = 'ssl/';
+
+    public $requestMap = [];
+
+    /**
+     * @param $domainName
+     * @return Recommendation
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getSslRecommendation($domainName)
```

Comment:
```suggestion
    public function getRecommendation($domainName)
```

This is the SSL Client using the SSL API, isn't this method name a bit redundant